### ISSUE TITLE
get_chat() caused struct.error: required argument is not an integer

### ISSUE THREAD

**Issue by dmitriyl** [LonamiWebs/Telethon#1382]
**Checklist**
* [V] The error is in the library's code, and not in my own.
* [V] I have searched for this issue before posting it and there isn't a duplicate.
* [V] I ran `pip install -U https://github.com/LonamiWebs/Telethon/archive/master.zip` and triggered the bug in the latest version.

**Code that causes the issue**
```python
from telethon import *
import logging

logging.basicConfig(format='[%(levelname) 5s/%(asctime)s] %(name)s: %(message)s', level=logging.WARNING)
app_id = *my-id*
api_hash = *my-hash*

client = TelegramClient('session_name', app_id, api_hash)

@client.on(events.NewMessage)
async def my_event_handler(event: events.newmessage.NewMessage.Event):
    chat: types.Chat = await event.get_chat()
    print(f"({chat.title}) - {event.raw_text}")


client.start()
client.run_until_disconnected()
...

```

**Traceback**
```
[ERROR/0000-00-00 00:00:00,000] telethon.client.updates: Unhandled exception on my_event_handler
Traceback (most recent call last):
  File "/Users/dmitry/PycharmProjects/telegram-parser/venv/lib/python3.7/site-packages/telethon/client/updates.py", line 428, in _dispatch_update
    await callback(event)
  File "/Users/dmitry/PycharmProjects/telegram-parser/app2.py", line 12, in my_event_handler
    chat: types.Chat = await event.get_chat()
  File "/Users/dmitry/PycharmProjects/telegram-parser/venv/lib/python3.7/site-packages/telethon/tl/custom/chatgetter.py", line 50, in get_chat
    await self._client.get_entity(self._input_chat)
  File "/Users/dmitry/PycharmProjects/telegram-parser/venv/lib/python3.7/site-packages/telethon/client/users.py", line 284, in get_entity
    functions.messages.GetChatsRequest(chats))).chats
  File "/Users/dmitry/PycharmProjects/telegram-parser/venv/lib/python3.7/site-packages/telethon/client/users.py", line 53, in __call__
    future = self._sender.send(request, ordered=ordered)
  File "/Users/dmitry/PycharmProjects/telegram-parser/venv/lib/python3.7/site-packages/telethon/network/mtprotosender.py", line 172, in send
    state = RequestState(request, self._loop)
  File "/Users/dmitry/PycharmProjects/telegram-parser/venv/lib/python3.7/site-packages/telethon/network/requeststate.py", line 17, in __init__
    self.data = bytes(request)
  File "/Users/dmitry/PycharmProjects/telegram-parser/venv/lib/python3.7/site-packages/telethon/tl/functions/messages.py", line 1146, in __bytes__
    b'\x15\xc4\xb5\x1c',struct.pack('<i', len(self.id)),b''.join(struct.pack('<i', x) for x in self.id),
  File "/Users/dmitry/PycharmProjects/telegram-parser/venv/lib/python3.7/site-packages/telethon/tl/functions/messages.py", line 1146, in <genexpr>
    b'\x15\xc4\xb5\x1c',struct.pack('<i', len(self.id)),b''.join(struct.pack('<i', x) for x in self.id),
struct.error: required argument is not an integer

```

**Comment by dmitriyl** [comment]
The code above works once in ten attempts, the rest of the time I get the error. When it works with, seems the line 'bytes(request)' is not called. Looks like something goes wrong with self._client.get_entity(self._input_chat) but I don't know how to check that. Any help please?

**Comment by dmitriyl** [comment]
It was fixed by 76fa7918a5979ab8362f13b9062a1d48ebaff0d4
